source/events/events.rst

```diff
@@ -101,52 +101,12 @@ An observatory Earth location should be given as well (see :ref:`coords-location
     * Version of the format (e.g. '1.0.0'). See :ref:`hduclass`.
 * ``HDUCLAS1`` type: string
     * Primary extension class (option: 'EVENTS'). See :ref:`hduclass`.
-* ``ORIGIN`` type: string
-    * Organisation that created the FITS file.
-* ``CREATOR`` type: string
-    * Software that created the file. When appropriate, the value of the
-      ``CREATOR`` keyword should also reference the specific version of the
-      program that created the FITS file. It is intented that this keyword
-      should refer to the program that originally defined the FITS file
-      structure and wrote the contents. If a FITS file is subsequently
-      copied largely intact into a new FITS by another program, then the value
-      of the ``CREATOR`` keyword should still refer to the original program.
-      ``HISTORY`` keywords should be used instead to document any further
-      processing that is performed on the file after it is created.
-      For more reading on the OGIP standard, see
-      `here <https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/ofwg_recomm/r7.html>`_.
-* ``TELESCOP`` type: string
-    * Telescope (e.g. 'CTA', 'HESS', 'VERITAS', 'MAGIC')
-* ``INSTRUME`` type: string
-    * Instrument used to aquire the data contained in the file
-      (e.g. 'North', 'South')
-* ``OBSERVER`` type: string
-    * Name of observer (e.g. 'Joe Public'). This could be for example the PI
-      of a proposal.
-* ``OBJECT`` type: string
-    * Observed object (e.g. 'Crab')
-* ``OBS_MODE`` type: string
-    * Observation mode (e.g. 'WOBBLE', 'SCAN', 'SLEW', or any mode that is
-      supported by ``TELESCOP``; string should be upper case)
 * ``OBS_ID`` type: int
     * Unique observation identifier (Run number)
 * ``TSTART`` type: float, unit: s
     * Start time of observation (relative to reference time, see :ref:`time`)
 * ``TSTOP`` type: float, unit: s
     * End time of observation (relative to reference time, see :ref:`time`)
-* ``EQUINOX`` type: float
-    * Equinox in years for the celestial coordinate system in which positions
-      given in either the header or data are expressed (options: 2000.0).
-      See also `HFWG Recommendation R3`_ for the OGIP standard.
-* ``RADECSYS`` type: string
-    * Stellar reference frame used for the celestial coordinate system in
-      which positions given in either the header or data are expressed.
-      (options: 'ICRS', 'FK5').
-      See also `HFWG Recommendation R3`_ for the OGIP standard.
-* ``TELAPSE`` type: float, unit: s
-    * Time interval between start and stop time (``TELAPSE=TSTOP-TSTART``).
-      Any gaps due to bad weather, or high background counts and/or other
-      anomalies, are included.
```

Comment:
No, telapse has nothing to do with WCS.

I used Githhub search on Gamamlib and ctools, and it looks like they use GTIs for exposure computation:
https://github.com/gammalib/gammalib/blob/e59bcbada35031caead880516d8b1daf806dd973/inst/cta/src/GCTACubeExposure.cpp#L853

Leaving `TELAPSE` in the `EVENTS` header also as completely redundant information only invites problems if some people start using it and relying on it.

In Gammapy we currently use `LIVETIME` in `EVENTS`, but plan to swtich to GTI soon (do the same as ctools)

Reply:
Sorry, I meant the 3 keywords above: EQUINOX, RADECSYS and TELAPSE.

I found [this](http://docs.astropy.org/en/stable/wcs/#validating-the-wcs-keywords-in-a-fits-file) after a quick search. Not sure if this makes the `RADECSYS` keyword required by wcs, but if it is, I would leave it. If it isn't, then +1 for it being optional.

Reply:
`EQUINOX` and `RADECSYS` were required before and still are. I did not change anything about them. They just appear in the diff because they were moved to a different place in the list of required keys.
I'm not sure about those, so defaulted to leave as-is.

Reply:
Ok, then forget what I said! I was assuming you moved also those.